.rubocop.yml

```diff
@@ -377,6 +377,14 @@ Style/AsciiComments:
 Style/DateTime:
   Enabled: false
 
+# This doesn't apply in routes.rb because Rails routes don't accept annotated
+# tokens (at least yet).  Also, enabling this produce a ton of new errors.
+# Let's talk about this before we enable it, I guess.
+Style/FormatStringToken:
+  Enabled: false
+  Exclude:
+    - "config/routes.rb"
```

Comment:
@pellaea Re configuration of `Style/FormatStringToken`
It shouldn't be producing **new** errors. I'll try to look at this tomorrow.